tools/manifest/vcs.py

```diff
@@ -61,24 +61,11 @@ def _local_changes(self):
         # type: () -> Set[bytes]
         """get a set of files which have changed between HEAD and working copy"""
         assert self.git is not None
-
-        changes = set()  # type: Set[bytes]
-
-        cmd = [b"status", b"-z", b"--ignore-submodules=all"]
-        data = self.git(*cmd)  # type: bytes
-
-        in_rename = False
-        for line in data.split(b"\0")[:-1]:
-            if in_rename:
-                changes.add(line)
-                in_rename = False
-            else:
-                status = line[:2]
-                if b"R" in status or b"C" in status:
-                    in_rename = True
-                changes.add(line[3:])
-
-        return changes
+        # note that git runs the command with tests_root as the cwd, which may
+        # not be the root of the git repo (e.g., within a browser repo)
+        cmd = [b"diff-index", b"--relative", b"--no-renames", b"--name-only", b"-z", b"HEAD"]
+        data = self.git(*cmd)
+        return set(data.split(b"\0"))
```

Comment:
Nit: I think this will always add an empty string to the set, which is probably harmless.

Reply:
It shouldn't; I did actually check to see if the null separation included the end.

Reply:
Cool! I was using `git diff -z` which did have a trailing null byte.